src/backends/caffe/caffelib.cc

```diff
@@ -870,12 +897,13 @@ namespace dd
       instantiate_template(ad);
     else // model template instantiation is defered until training call
       {
+
 	update_deploy_protofile_softmax(ad); // temperature scaling
 	create_model(true);
       }
 
     // the first present measure will be used to snapshot best model
-    _best_metrics = {"map", "meaniou", "mlacc", "delta_score_0.1", "bacc", "f1", "net_meas", "acc"};
+    _best_metrics = {"map", "meaniou", "mlacc", "delta_score_0.1", "bacc", "f1", "net_meas", "acc", "eucll"};
```

Comment:
You are using L1 loss most often as it seems. Do we want `eucll` as best metrics here ?

Reply:
L1 metric is not yet implemented ...

Reply:
putting in my todo list

Reply:
Agreed to have a dedicated PR to it.